Source/CombatExtended/CombatExtended/Verbs/Verb_MeleeAttackCE.cs

```diff
@@ -409,7 +415,8 @@ private void DoParry(Pawn defender, Thing parryThing, bool isRiposte = false)
                 if (parryThing is Apparel_Shield)
                 {
                     // Shield bash
-                    DamageInfo dinfo = new DamageInfo(DamageDefOf.Blunt, 6, -1, defender, null, parryThing.def);
+                    DamageInfo dinfo = new DamageInfo(DamageDefOf.Blunt, 6, 0, //Armor Penetration
```

Comment:
AP 0 would disable shield bash damage. Should either use default value or get it from shield (better solution but don't know off-hand how difficult to implement). Also I think we found a previous version bug here since I don't think shield AP is ever actually handled in B18.

Reply:
Looking at here, 0 seems to be the default value:
https://github.com/josh-m/RW-Decompile/blob/master/Verse/DamageInfo.cs#L172

Reply:
Default value as in one value to use for all shields. DamageInfo defaults to 0 but the way CE armor works it'll just get completely deflected by anything.

Reply:
*Should* be fixed.